NohanaImagePicker/AssetListSelectableDateSectionController.swift

```diff
@@ -332,12 +331,25 @@ extension AssetListSelectableDateSectionController: AssetCellDelegate {
 }
 
 // MARK: - PhotoAuthorizationLimitedCellDeletate
+
 extension AssetListSelectableDateSectionController: PhotoAuthorizationLimitedCellDeletate {
     func didSelectAddPhotoButton(_ cell: PhotoAuthorizationLimitedCell) {
-
+        NotificationCenter.default.post(
+            Notification(
+                name: NotificationInfo.PhotoAuthorization.didTapAddPhoto,
+                object: nil,
+                userInfo: nil
+            )
+        )
```

Comment:
`NohanaImagePickerControllerDelegate` can run `nohanaImagePickerController.delegate` property in `AssetListSelectableDateSectionController`.

So, Please run `nohanaImagePickerController.delegate` instead of `NotificationCenter`